submissions/NadiaVorontsova/hooli-style-popup/style.css

```diff
@@ -0,0 +1,144 @@
+*{
```

Comment:
please, use space after each css-selector
```suggestion
* {
```

Reply:
I fixed it